src/containers/About/Issues.js

```diff
@@ -29,22 +30,28 @@ class Issues extends React.Component {
       sort: "created",
       labels: "",
       assignee: ""
-    }
+    },
+    initialized: false,
+    loadingSignal: false
```

Comment:
The meaning of `Signal` is not clear and also redundent.
`loading` or `isLoading` are more clear